### PR TITLE
Events filters: format, interpretations and audiences

### PR DESCRIPTION
**Pull request by rcantin-w**
## Who is this for?
Events work
Part of #10647

## What is it doing for them?
- Adds the format, interpretation and audience filters, which are the most straightforward ones and ready to go behind the toggle.
- Starts to add a disabled style to the checkbox, it'll get reviewed as part of [adding the "is available online" boolean filter.](https://github.com/wellcomecollection/wellcomecollection.org/issues/10668)

### CODE REVIEW

File: common/views/components/CheckboxRadio/CheckboxRadio.tsx [hunk 1]
```diff
@@ -59,6 +64,15 @@ const CheckboxRadioInput = styled.input.attrs<{ $type: string }>(props => ({
     border-width: 2px;
   }
 
+  &[disabled] {
+    & ~ ${CheckboxRadioBox} {
+      color: pink;
```
Comment (davidpmccormick) [inline common/views/components/CheckboxRadio/CheckboxRadio.tsx:69]: 💅

Comment (rcantin-w) [reply common/views/components/CheckboxRadio/CheckboxRadio.tsx:69]: Oops. But it _is_ Wednesday.

Comment (rcantin-w) [reply common/views/components/CheckboxRadio/CheckboxRadio.tsx]: I removed the changes, we can address those again when we actually add a single-checkbox filter.

File: content/webapp/pages/search/events.tsx [hunk 2]
```diff
@@ -56,13 +60,52 @@ const SortPaginationWrapper = styled.div`
...
   const hasNoResults = eventResponseList.totalResults === 0;
+  const hasActiveFilters = hasFilters({
+    filters: filters.map(f => f.id),
+    queryParams: query,
+  });
 
   return (
     <Space $v={{ size: 'l', properties: ['padding-bottom'] }}>
+      {(!hasNoResults || (hasNoResults && hasActiveFilters)) && (
+        <>
```
Comment (davidpmccormick) [inline content/webapp/pages/search/events.tsx:81]: is this extra `Fragment` needed?

**Review by davidpmccormick — APPROVED** [review]
Nice!

**Pull request merged**